nginx.conf

```diff
@@ -13,13 +13,104 @@
 #
 # One difference between this file and the configuration for production is that
 # this configuration does not support request body decompression.
+daemon off;
+worker_processes 1;
+error_log stderr;
+pid ./tmp/nginx.pid;
```

Comment:
Are these changes needed? I think it's really helpful to minimize the number of differences between this file and the nginx configuration in [prod](https://github.com/getodk/central/blob/master/files/nginx/odk.conf.template).

Reply:
`daemon off`: this means that nginx will run as a foreground process, which seems a lot less surprising for frontend devs not familiar with nginx

`worker_processes 1`: this is the default, so can be removed

`error_log stderr`: i think this is both necessary and helpful - it removes one point of OS-specificity, and means that devs will see nginx errors in their terminal rather than have to tail the error log file

`pid ./tmp/nginx.pid`: again this removes another place where nginx may default to an absolute path which the current user does not have access to